Frameworks/Foundation/NSOperation.mm

```diff
@@ -77,41 +61,88 @@ + (BOOL)automaticallyNotifiesObserversForKey:(NSString*)key {
     return NO;
 }
 
-/**
- @Status Interoperable
-*/
-+ (id)allocWithZone:(NSZone*)zone {
-    NSOperation* ret = [super allocWithZone:zone];
+- (void)_checkReady {
```

Comment:
Needs another lock

---
In reply to: [69623776](https://github.com/Microsoft/WinObjC/pull/602#discussion_r69623776) [](ancestors = 69623776)